docs/src/components/App.js

```diff
@@ -30,7 +30,7 @@ export default function App({ children }: Props): Node {
 
                   <Divider />
 
-                  <Box width="100%">
+                  <Box width="100%" minWidth={0}>
```

Comment:
![image](https://user-images.githubusercontent.com/127199/115477759-7f52e600-a1f9-11eb-887e-5eef3a0b9fac.png)

More information at https://css-tricks.com/flexbox-truncated-text/#the-solution-is-min-width-0-on-the-flex-child

Reply:
Gahhh I swear this one thing has been the source of every other bug when it comes to Flex issues - great find!

Reply:
Hm…is this a universal enough fix that we should add it as a hardcoded attribute on FlexItem?

Reply:
@dangerismycat I like that idea, to add `minWidth` as a prop to `Flex.Item` and perhaps add a link to that article. I wouldn't make it the default though since it goes against the Flex spec.